scripts/dts/gen_defines.py

```diff
@@ -600,9 +600,12 @@ def out(ident, val, aliases=()):
     # Also writes any aliases listed in 'aliases' (an iterable). For the
     # header, these look like '#define <alias> <ident>'. For the configuration
     # file, the value is just repeated as '<alias>=<val>' for each alias.
+    #
+    # Returns the generated macro name for ident.
```

Comment:
(Could make it 'ident' just for consistency. Idea is to make it clearer that it's an argument name and not a shortening.)